<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchUnit</class>
 <widget class="QWidget" name="SearchUnit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>154</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>154</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>154</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>搜索</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>4</y>
     <width>61</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioButton_id">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>编号</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_rowcol">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>行列</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>4</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>编号：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_id"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>行  ：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_row"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>列  ：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_col"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>导入</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
